docs/design/README.md

```diff
@@ -1223,25 +1222,6 @@ fn Foo() {
 > -   Proposal
 >     [#162: Basic Syntax](https://github.com/carbon-language/carbon-lang/pull/162)
 
-### Assignment statements
```

Comment:
Any reason to not keep at least a brief overview here?

Reply:
I do think there should be something inside the previous section about assignment statements if this section is going to be removed, like around the sentence starting with "Expressions."

Reply:
Makes sense, done.